Many link fixes, replace full-stop in file names with hyphen

ctzsnooze opened the pull request:
In trying to get a redirect to work, I found that `yarn build` gave a much better link check than `yarn start`.

It found many more failed links, and there were a lot.  However as @haslinghuis noted, even though `yarn build` might report zero link errors, it still misses a lot.

Docusauris is VERY FUSSY about links:

- links like `(docs/dir/file)` generally work, but randomly don't.  Always prepend `/` before `/docs` to ensure the link is properly formed; otherwise sometimes (randomly), `docs/something` is assumed to be a relative link.
- periods (full-stops) in file names cause issues and must to be avoided, use hyphens instead
- llink names must be capitalised the exact same way as the file name, or they won't work.
- because .md files are converted to .html, it's better to not include the extension `.md` in the link syntax, it isn't needed.
- relative links can be used if linking to a file further out in the document hierarchy.  eg if you're in `/docs/dir/myfile` and the link is to `/docs/dir/nextDir/otherfile`, you can use `nextDir/otherFile` as the link.
- relative links to files in the same directory should either use the full path from `/docs`, including the slash, ie `/docs/dir/A.md`, or a simple relative link, ie [my link](A), nothing else.  A full link is more portable if the whole document is moved around later.
- take care setting the ID of a file, at the top inside `---` characters, since that will become name used for links (it will over-ride the file name.
- where a link is to a directory containing only files, and with a 'category.js` file in the directory, the syntax is `/docs/contents/dirName`.  Don't forget the `/` at the start.

Fixes:
- .md terminators in linked file names
- capitalise links to match file names
- user relative links from within same directory and to subdirectories
- keep links local where possible, removed non-local links to betaflight.com
- used `/docs/` throughout
- fixed the category link behaviour for tuning and release notes.

There are zero link errors at present.

I was unable to implement a redirect mechanism.  `chatAi` thought it should be simple and gave specific instructions, but none of the ideas it floated appeared to work on a built site.

This includes a revert for the previous redirect attempt PR #301.

haslinghuis commented on docs/development/Safety.md:
```diff
@@ -7,14 +7,14 @@ As many can attest, multirotors and RC models in general can be very dangerous,
 ## Before Installing
 
-Please consult the [Cli](Cli.md), [Controls](Controls.md), [Failsafe](/docs/wiki/guides/current/Failsafe.md) and [Modes](Modes.md)
+Please consult the [Cli](Cli.md), [Controls](Controls.md), [Failsafe](docs/wiki/guides/current//Failsafe.md) and [Modes](Modes.md)
```
```suggestion
Please consult the [Cli](Cli.md), [Controls](Controls.md), [Failsafe](docs/wiki/guides/current/Failsafe.md) and [Modes](Modes.md)
```

ctzsnooze replied:
fixed

haslinghuis commented on docs/development/Safety.md:
```diff
@@ -7,14 +7,14 @@ As many can attest, multirotors and RC models in general can be very dangerous,
 are reached when controls are operated. Failure to configure these ranges properly can create
 problems, such as inability to arm (because you can't reach the endpoints) or immediate activation of
-[failsafe](/docs/wiki/guides/current/Failsafe.md).
+[failsafe](docs/wiki/guides/current//Failsafe.md).
```
```suggestion
[failsafe](docs/wiki/guides/current/Failsafe.md).
```

ctzsnooze replied:
done

haslinghuis commented on docs/wiki/boards/Missing.md:
```diff
@@ -2,6 +2,6 @@
+You can try to find a previous board / board version from same Manufacturer.
 
-You can ask the Manufacturer to add this description as described in [https://betaflight.com/docs/manufacturer/manufacturer-design-guidelines](https://betaflight.com/docs/manufacturer/manufacturer-design-guidelines)
+You can ask the Manufacturer to add this description as described in [anufacturer design guidelines](docs/development/manufacturer/manufacturer-design-guidelines)
```
```suggestion
You can ask the Manufacturer to add this description as described in the [Manufacturer Design Guidelines](docs/development/manufacturer/manufacturer-design-guidelines)
```

haslinghuis commented on docs/wiki/boards/OMNIBUS.md:
```diff
@@ -60,7 +60,7 @@ _(add your name here if you help test or contribute code for this board)_
 _(add links board descriptions here that are similar in features or function, but have a separate target)_
 
-- [SirinFPV](Board---SIRINFPV)
+- [SirinFPV](SIRINFPV)
```
Should this be?
```suggestion
- [SirinFPV](doc/wiki/boards/SIRINFPV)
```

haslinghuis commented on docs/wiki/boards/PIKOBLX.md:
```diff
@@ -1,3 +1,3 @@
 #### This board target has been renamed to FF_PIKOBLX.
 
-[FF_PIKOBLK board wiki page](Board---FF_PIKOBLX)
+[FF_PIKOBLK board wiki page](FF_PIKOBLX)
```
```suggestion
[FF_PIKOBLK board wiki page](docs/wiki/boards/FF_PIKOBLX)
```

ctzsnooze replied:
No, this is how a link must be formatted when the file is at the same directory level.  Otherwise the link will fail when built.

haslinghuis commented on docs/wiki/boards/SIRINFPV.md:
```diff
@@ -59,7 +59,7 @@ _(add your name here if you help test or contribute code for this board)_
 _(add links board descriptions here that are similar in features or function, but have a separate target)_
 
-- [OMNIBUS](Board---OMNIBUS)
+- [OMNIBUS](OMNIBUS)
```
```suggestion
- [OMNIBUS](docs/wiki/boards/OMNIBUS)
```

ctzsnooze replied:
see above

haslinghuis commented on docs/wiki/configurator/blackbox-tab.md:
```diff
@@ -8,4 +8,4 @@ sidebar_position: 20
 :::
 
-For more information about Blackbox, see the [Wiki document](/docs/wiki/guides/current/Failsafe.md).
+For more information about Blackbox, see the [Wiki document](docs/wiki/guides/current//Failsafe.md).
```
```suggestion
For more information about Blackbox, see the [Wiki document](docs/wiki/guides/current/Failsafe.md).
```

ctzsnooze replied:
fixed, no need to repeat

haslinghuis commented on docs/wiki/configurator/failsafe-tab.md:
```diff
@@ -8,4 +8,4 @@ sidebar_position: 5
 :::
 
-For more information about Failsafe, see the [Wiki document](/docs/wiki/guides/current/GPS-Rescue-v4-5.md).
+For more information about Failsafe, see the [Wiki document](docs/wiki/guides/current//GPS-Rescue-v4-5.md).
```
```suggestion
For more information about Failsafe, see the [Wiki document](docs/wiki/guides/current/GPS-Rescue-v4-5.md).
```

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/configurator/gps-tab.md:
```diff
@@ -8,4 +8,4 @@ sidebar_position: 12
 :::
 
-For more information about GPS changes in 4.5, see the [Wiki document](/docs/wiki/guides/current/Failsafe.md).
+For more information about GPS changes in 4.5, see the [Wiki document](docs/wiki/guides/current//Failsafe.md).
```
```suggestion
For more information about GPS changes in 4.5, see the [Wiki document](docs/wiki/guides/current/Failsafe.md).
```

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/configurator/vtx-tab.md:
```diff
@@ -10,5 +10,5 @@ Sorry; this Wiki page has not yet been written.
 
-- VTx Tables, see this [Wiki document](/docs/wiki/guides/current/VTX-Tables.md).
-- VTx CLI commands, see this [Wiki document](/docs/wiki/guides/current/VTX-CLI-Settings.md).
+- VTx Tables, see this [Wiki document](docs/wiki/guides/current//VTX-Tables.md).
+- VTx CLI commands, see this [Wiki document](docs/wiki/guides/current//VTX-CLI-Settings.md).
```
```suggestion
- VTx Tables, see this [Wiki document](docs/wiki/guides/current/VTX-Tables.md).
- VTx CLI commands, see this [Wiki document](docs/wiki/guides/current/VTX-CLI-Settings.md).
```

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/guides/current/ESC-Telemetry.md:
```diff
@@ -17,9 +17,9 @@ JoeLucid, in March 2019, developed the bidirectional DShot code to send RPM data
-- [RPM filtering](/docs/wiki/guides/current/DSHOT-RPM-Filtering.md), where dynamic notch filters tracked each motor's center frequency - 12 dynamic RPM filters for a quadcopter - greatly improving RPM based noise rejection
-- [Dynamic Idle](/docs/wiki/guides/current/Dynamic-Idle.md), where the drive to the motors can be rapidly changed, under PID control, so that a motor's RPM does not fall below a defined minimum RPM value. Compared to a simple minimum idle throttle value, this reduces the likelihood of a desync and allows the PIDs to use the full range of motor drive values.
-- [RPM Limiting](/docs/wiki/release/betaflight-4.5-release-notes#161-rpm-limiter-build-option), which allows the user to limit maximum average RPM, for spec class racing.
+- [RPM filtering](docs/wiki/guides/current//DSHOT-RPM-Filtering.md), where dynamic notch filters tracked each motor's center frequency - 12 dynamic RPM filters for a quadcopter - greatly improving RPM based noise rejection
+- [Dynamic Idle](docs/wiki/guides/current//Dynamic-Idle.md), where the drive to the motors can be rapidly changed, under PID control, so that a motor's RPM does not fall below a defined minimum RPM value. Compared to a simple minimum idle throttle value, this reduces the likelihood of a desync and allows the PIDs to use the full range of motor drive values.
```
```suggestion
- [RPM filtering](docs/wiki/guides/current/DSHOT-RPM-Filtering.md), where dynamic notch filters tracked each motor's center frequency - 12 dynamic RPM filters for a quadcopter - greatly improving RPM based noise rejection
- [Dynamic Idle](docs/wiki/guides/current/Dynamic-Idle.md), where the drive to the motors can be rapidly changed, under PID control, so that a motor's RPM does not fall below a defined minimum RPM value. Compared to a simple minimum idle throttle value, this reduces the likelihood of a desync and allows the PIDs to use the full range of motor drive values.
```

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/guides/current/FAQ.md:
```diff
@@ -563,7 +563,7 @@ Take a look at this overview as it describes how this should be done: http://www
 Some additional information can be found from 6:20 onwards in this video: http://www.youtube.com/watch?v=htkw7d97bOo
 
-NOTE: Failsafe configuration has changed in Betaflight 2.4.0 onwards and CF Configurator 1.2.0. The relevant documentation can be found [here](/docs/wiki/guides/current/Failsafe.md).
+NOTE: Failsafe configuration has changed in Betaflight 2.4.0 onwards and CF Configurator 1.2.0. The relevant documentation can be found [here](docs/wiki/guides/current//Failsafe.md).
```
```suggestion
NOTE: Failsafe configuration has changed in Betaflight 2.4.0 onwards and CF Configurator 1.2.0. The relevant documentation can be found [here](docs/wiki/guides/current/Failsafe.md).
```

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/release/Betaflight-4-1-Release-Notes.md:
```diff
@@ -28,8 +28,8 @@ _Have fun_
 - version 1.4.0 of the [Betaflight TX lua scripts](https://github.com/betaflight/betaflight-tx-lua-scripts/releases) has been released. This includes changes to go with Betaflight 4.1 (installation instructions [here](https://github.com/betaflight/betaflight-tx-lua-scripts#installing));
-- bidirectional Dshot, which is the basis for RPM based filtering, has been improved and is now available with BLHeli\_32 (from version 32.7 on) and on BLHeli\_S hardware (by using the JESC firmware). Use [these instructions](/docs/wiki/guides/current/DSHOT-RPM-Filtering) to get it set up ([#8554](https://github.com/betaflight/betaflight/pull/8554), [#8779](https://github.com/betaflight/betafligh ...
-- as you have come to expect, there is a detailed [Tuning Notes](/docs/wiki/tuning/4.1-Tuning-Notes) for Betaflight 4.1. Use them, or use the new tuning sliders in the Betaflight configurator 10.6.0 to get your craft tuned. Please **do not paste tuning configurations from previous versions of the firmware**. Some defaults have been changed, and some parameters are used in different ways, so previ ...
+- bidirectional Dshot, which is the basis for RPM based filtering, has been improved and is now available with BLHeli\_32 (from version 32.7 on) and on BLHeli\_S hardware (by using the JESC firmware). Use [these instructions](docs/wiki/guides/current//DSHOT-RPM-Filtering) to get it set up ([#8554](https://github.com/betaflight/betaflight/pull/8554), [#8779](https://github.com/betaflight/betafligh ...
```
```suggestion
- bidirectional Dshot, which is the basis for RPM based filtering, has been improved and is now available with BLHeli\_32 (from version 32.7 on) and on BLHeli\_S hardware (by using the JESC firmware). Use [these instructions](docs/wiki/guides/current/DSHOT-RPM-Filtering) to get it set up ([#8554](https://github.com/betaflight/betaflight/pull/8554), [#8779](https://github.com/betaflight/betaflight/pull/8779));
```

ctzsnooze replied:
done

haslinghuis commented on docs/wiki/release/Betaflight-4-4-Release-Notes.md:
```diff
@@ -98,7 +98,7 @@ Favorite presets are being remembered by it's path+name in the repo. So a favori
 **GPS "Rescue" has been extensively revised and greatly improved.** The quad should reliably return at the set speed, descend at an angle, land within a few metres of the home point, and disarm automatically on touch-down. There are separate PID control elements for altitude and return velocity t home; the defaults work very well for 'typical' quads. The system should initially be tested with a s ...
 
-**We strongly recommend reading the [instructions](/docs/wiki/guides/current/GPS-Rescue-v4-4).**
+**We strongly recommend reading the [instructions](docs/wiki/guides/current//GPS-Rescue-v4-4).**
```
```suggestion
**We strongly recommend reading the [instructions](docs/wiki/guides/current/GPS-Rescue-v4-4).**
```

ctzsnooze replied:
done

haslinghuis commented on docs/wiki/release/Betaflight-4-5-Release-Notes.md:
```diff
@@ -66,7 +66,7 @@ When the FC boots, our UBLox code cycles through all available baud rates on the
 If the FC is connected to Configurator at boot time, we request the full satellite information list, so that we can populate the detailed satellite information list on the left side of Configurator's GPS tab.  Otherwise this information is not requested, because we do not require it while in flight, and it adds a lot of serial port traffic when enabled.
 
-The CPU cost and task timing for GPS data has been extensively reviewed and optimised.  Even so, GPS Rescue puts a huge load on a CPU.  For reliability it is best to use a 4k PID loop on most processors, especially at 57600 baud.  More information about CPU load vs Baud Rate is available in the [GPS Rescue 4.5 documentation](/docs/wiki/guides/current/GPS-Rescue-v4-5).  The CLI `tasks` command may ...
+The CPU cost and task timing for GPS data has been extensively reviewed and optimised.  Even so, GPS Rescue puts a huge load on a CPU.  For reliability it is best to use a 4k PID loop on most processors, especially at 57600 baud.  More information about CPU load vs Baud Rate is available in the [GPS Rescue 4.5 documentation](docs/wiki/guides/current//GPS-Rescue-v4-5).  The CLI `tasks` command may ...
```
```suggestion
The CPU cost and task timing for GPS data has been extensively reviewed and optimised.  Even so, GPS Rescue puts a huge load on a CPU.  For reliability it is best to use a 4k PID loop on most processors, especially at 57600 baud.  More information about CPU load vs Baud Rate is available in the [GPS Rescue 4.5 documentation](docs/wiki/guides/current/GPS-Rescue-v4-5).  The CLI `tasks` command may be used to check CPU usage and task over-runs when evaluating the impact of baud rate in relation to PID loop frequency.
```

ctzsnooze replied:
done

haslinghuis commented on docs/wiki/release/older/Betaflight-3-0-Release-Notes.md:
```diff
@@ -457,7 +457,7 @@ Clear as mud?
 ### Notch Filters
 
-See: [Black-Box logging and usage](/wiki/guides/current/Black-Box-logging-and-usage)
+See: [Black-Box logging and usage](docs/wiki/guides/current//Black-Box-logging-and-usage)
```
```suggestion
See: [Black-Box logging and usage](docs/wiki/guides/current/Black-Box-logging-and-usage)
```

ctzsnooze replied:
done

haslinghuis commented on docs/wiki/release/older/Betaflight-3-4-Release-Notes.md:
```diff
@@ -6,13 +6,13 @@ sidebar_position: 10
+For pilots who are into long range flying, we have added the brand new ['GPS Rescue' mode](docs/wiki/guides/archive/GPS-Rescue-Mode). It is a simplified version of the 'return to home' mode that exists in other, more navigation oriented firmware, and the great news is that all it needs to work is a GPS, no fiddling to make a compass work required!
 
-Last but not least, we've also added a bunch of new features to improve the convenience of using Betaflight: You can now copy / paste your logs from an SD card or the on board flash chip by [mounting the flight controller as a storage device](/docs/wiki/guides/current/Mass-Storage-Device-Support), and you can use your flight controller / TX to [emulate a joystick](/docs/wiki/guides/current/HID-Jo ...
+Last but not least, we've also added a bunch of new features to improve the convenience of using Betaflight: You can now copy / paste your logs from an SD card or the on board flash chip by [mounting the flight controller as a storage device](docs/wiki/guides/current//Mass-Storage-Device-Support), and you can use your flight controller / TX to [emulate a joystick](docs/wiki/guides/current//HID-Jo ...
```
```suggestion
Last but not least, we've also added a bunch of new features to improve the convenience of using Betaflight: You can now copy / paste your logs from an SD card or the on board flash chip by [mounting the flight controller as a storage device](docs/wiki/guides/current//Mass-Storage-Device-Support), and you can use your flight controller / TX to [emulate a joystick](docs/wiki/guides/current/HID-Joystick-Support) with no extra hardware needed, in order to fly on a simulator.
```

ctzsnooze replied:
done

haslinghuis commented on docs/wiki/release/older/Betaflight-3-4-Release-Notes.md:
```diff
@@ -55,7 +55,7 @@ Happy Props!
 - Added support for throttle limiting ([#5608](https://github.com/betaflight/betaflight/pull/5608));
 - Added PID loop improvements ([#5968](https://github.com/betaflight/betaflight/pull/5968), [#5963](https://github.com/betaflight/betaflight/pull/5963), [#5962](https://github.com/betaflight/betaflight/pull/5962));
-- Added support for accelerated [yaw spin recovery](/docs/wiki/guides/current/Yaw-Spin-Recovery-and-Gyro-Overflow-Detect) ([#5706](https://github.com/betaflight/betaflight/pull/5706));
+- Added support for accelerated [yaw spin recovery](docs/wiki/guides/current//Yaw-Spin-Recovery-and-Gyro-Overflow-Detect) ([#5706](https://github.com/betaflight/betaflight/pull/5706));
```
```suggestion
- Added support for accelerated [yaw spin recovery](docs/wiki/guides/current/Yaw-Spin-Recovery-and-Gyro-Overflow-Detect) ([#5706](https://github.com/betaflight/betaflight/pull/5706));
```

ctzsnooze replied:
done

haslinghuis commented on docs/wiki/tuning/4-0-Tuning-Notes.md:
```diff
@@ -4,17 +4,17 @@ sidebar_position: 13
-- [Transient throttle limit](4.0-Tuning-Notes#transient-throttle-limit), which improves noise behaviour at low and high throttle by preventing refected noise from doubling up and becoming distorted
-- [Improved yaw PIDs](4.0-Tuning-Notes#improved-yaw-pids), with a better understanding of how to tune yaw
-- [Improved dynamic notch code](4.0-Tuning-Notes#improved-dynamic-notch-code) has been improved to better track and suppress motor noise with less delay, using two closely spaced, narrower notches
+- [RPM controlled multiple dynamic notch filtering over bi-directional DShot](docs/wiki/guides/current//DSHOT-RPM-Filtering), to markedly improve motor noise suppression, reducing filter delay time
+- [Launch Control](docs/wiki/guides/current//Launch-Control), allows a pilot to hold a specified angle precisely and accurately for the perfect race launch
+- [D_min](docs/wiki/guides/current//DMIN.md), which dynamically adjusts the PID D parameter according to stick movement
+- [Dynamic lowpass filtering](#dynamic-lowpass-filtering), to improve noise control at low rpm and reduce filter delay time at high rpm
+- [Improved dynamic notch code](#improved-dynamic-notch-code), with better tracking and better noise rejection with less delay,
+- [D-only TPA](#d-only-tpa), to selectively reduce D related noise at high throttle and maintain normal P responsiveness
+- [Integrated yaw](docs/wiki/guides/current//Integrated-Yaw), an experimental option which mathematically integrates the yaw PID values, potentially simplifying yaw tuning
```
```suggestion
- [RPM controlled multiple dynamic notch filtering over bi-directional DShot](docs/wiki/guides/current/DSHOT-RPM-Filtering), to markedly improve motor noise suppression, reducing filter delay time
- [Launch Control](docs/wiki/guides/current/Launch-Control), allows a pilot to hold a specified angle precisely and accurately for the perfect race launch
- [D_min](docs/wiki/guides/current/DMIN.md), which dynamically adjusts the PID D parameter according to stick movement
- [Dynamic lowpass filtering](#dynamic-lowpass-filtering), to improve noise control at low rpm and reduce filter delay time at high rpm
- [Improved dynamic notch code](#improved-dynamic-notch-code), with better tracking and better noise rejection with less delay,
- [D-only TPA](#d-only-tpa), to selectively reduce D related noise at high throttle and maintain normal P responsiveness
- [Integrated yaw](docs/wiki/guides/current/Integrated-Yaw), an experimental option which mathematically integrates the yaw PID values, potentially simplifying yaw tuning
```

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/tuning/4-0-Tuning-Notes.md:
```diff
@@ -294,7 +294,7 @@ The challenge with filtering is to remove as much noise as possible above the cu
 Most noise arises from the rotation of each motor, which shakes the frame a little bit every rotation. This generates an rpm-dependent noise signal at the fundamental frequency of the motor (rpm \* 60 in hz), and secondary integral multiples of that frequency (harmonics). All propeller blades also have natural resonant frequencies, and can abruptly and suddenly generate bad oscillations at their  ...
 
-Using bi-directional DShot, and a BLHeli32 ESC with suitable code, the new [betaflight rpm notch filters](/docs/wiki/guides/current/DSHOT-RPM-Filtering) identify the fundamental and harmonic frequencies of every motor, and target the related noise peaks with very narrow notch filters that almost totally remove motor-frequency-related noise. Usually less filtering is required, resulting in about h ...
+Using bi-directional DShot, and a BLHeli32 ESC with suitable code, the new [betaflight rpm notch filters](docs/wiki/guides/current//DSHOT-RPM-Filtering) identify the fundamental and harmonic frequencies of every motor, and target the related noise peaks with very narrow notch filters that almost totally remove motor-frequency-related noise. Usually less filtering is required, resulting in about h ...
```
```suggestion
Using bi-directional DShot, and a BLHeli32 ESC with suitable code, the new [betaflight rpm notch filters](docs/wiki/guides/current/DSHOT-RPM-Filtering) identify the fundamental and harmonic frequencies of every motor, and target the related noise peaks with very narrow notch filters that almost totally remove motor-frequency-related noise. Usually less filtering is required, resulting in about half to two-thirds the filter delay of 'standard' filtering. credit: JoeLucid.
```

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/tuning/4-0-Tuning-Notes.md:
```diff
@@ -469,7 +469,7 @@ credit: JoeLucid
 
 :::note
-The following notes apply ONLY to un-integrated yaw configuration! If you're flying [integrated yaw](/docs/wiki/guides/current/Integrated-Yaw), the tuning process is completely different!
+The following notes apply ONLY to un-integrated yaw configuration! If you're flying [integrated yaw](docs/wiki/guides/current//Integrated-Yaw), the tuning process is completely different!
```
```suggestion
The following notes apply ONLY to un-integrated yaw configuration! If you're flying [integrated yaw](docs/wiki/guides/current/Integrated-Yaw), the tuning process is completely different!
```

ctzsnooze replied:
see below

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/tuning/4-1-Tuning-Notes.md:
```diff
@@ -33,13 +33,13 @@ We strongly recommend enabling RPM based filtering. This is now easier than ever
+For a VTX to be controlled by betaflight, [a suitable VTX table must be configured by the user](docs/development/VTX.md#vtx-table) in a way that complies with local regulations.
 
-Bitbanged Dshot is very new; please report bugs during the RC phase to GitHub. If you have an issue with your ESC, try `set dshot_bitbang = OFF` rather than AUTO. For more information see the [rpm telemetry page](/docs/wiki/guides/current/DSHOT-RPM-Filtering)
+Bitbanged Dshot is very new; please report bugs during the RC phase to GitHub. If you have an issue with your ESC, try `set dshot_bitbang = OFF` rather than AUTO. For more information see the [rpm telemetry page](docs/wiki/guides/current//DSHOT-RPM-Filtering)
```
```suggestion
Bitbanged Dshot is very new; please report bugs during the RC phase to GitHub. If you have an issue with your ESC, try `set dshot_bitbang = OFF` rather than AUTO. For more information see the [rpm telemetry page](docs/wiki/guides/current/DSHOT-RPM-Filtering)
```

ctzsnooze replied:
see below

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/tuning/4-1-Tuning-Notes.md:
```diff
@@ -33,13 +33,13 @@ We strongly recommend enabling RPM based filtering. This is now easier than ever
 ## Simpler RPM Filter Configuration
 
-Enabling [rpm based filtering](/docs/wiki/guides/current/DSHOT-RPM-Filtering) is now easier than before. With the default `dshot_bitbang = AUTO`, there is no need for custom timer and DMA changes on F4 and F7 boards. RPM telemetry will be automatically activated when `dshot_bidir` is enabled, and this can be done in the 10.6 Configurator. The older method is active when `dshot_bitbang = OFF`. For ...
+Enabling [rpm based filtering](docs/wiki/guides/current//DSHOT-RPM-Filtering) is now easier than before. With the default `dshot_bitbang = AUTO`, there is no need for custom timer and DMA changes on F4 and F7 boards. RPM telemetry will be automatically activated when `dshot_bidir` is enabled, and this can be done in the 10.6 Configurator. The older method is active when `dshot_bitbang = OFF`. For ...
```
```suggestion
Enabling [rpm based filtering](docs/wiki/guides/current/DSHOT-RPM-Filtering) is now easier than before. With the default `dshot_bitbang = AUTO`, there is no need for custom timer and DMA changes on F4 and F7 boards. RPM telemetry will be automatically activated when `dshot_bidir` is enabled, and this can be done in the 10.6 Configurator. The older method is active when `dshot_bitbang = OFF`. For more information check the updated rpm_telemetry page.
```

ctzsnooze replied:
see below

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/tuning/4-2-Tuning-Notes.md:
```diff
@@ -6,37 +6,37 @@ sidebar_position: 11
 **NOTE 4**: If using the Accelerometer, arming will fail unless it has been calibrated.
 
-**NOTE 5**: We recommend using eRPM based filtering. Read about [enabling and configuring rpm filtering here.](/docs/wiki/guides/current/DSHOT-RPM-Filtering)
+**NOTE 5**: We recommend using eRPM based filtering. Read about [enabling and configuring rpm filtering here.](docs/wiki/guides/current//DSHOT-RPM-Filtering)
```
```suggestion
**NOTE 5**: We recommend using eRPM based filtering. Read about [enabling and configuring rpm filtering here.](docs/wiki/guides/current/DSHOT-RPM-Filtering)
```

ctzsnooze replied:
Searched and fixed all similar ones, thanks for the heads up

ctzsnooze replied:
fixed

haslinghuis commented on docs/wiki/tuning/older/3-4-tuning-notes.md:
```diff
@@ -37,9 +33,9 @@ This removes spikes and sharp edges that otherwise arise during rapid stick inpu
 If you enable item_relax, throttle_boost, or use D setpoint weight above zero, you definitely should activate RC smoothing on the relevant axes, as above.
 
-_IMPORTANT_: SBus or FPort users [should stop using OpenTX when using 16 channels](SBus-FPort-and-RC-Smoothing-when-using-OpenTX).
+_IMPORTANT_: SBus or FPort users [should stop using OpenTX when using 16 channels](docs/wiki/guides/current//SBus-FPort-and-RC-Smoothing-when-using-OpenTX).
```
```suggestion
_IMPORTANT_: SBus or FPort users [should stop using OpenTX when using 16 channels](docs/wiki/guides/current/SBus-FPort-and-RC-Smoothing-when-using-OpenTX).
```

ctzsnooze replied:
Thanks, I missed those, and they were not detected during build.  Found them all with search and replace, are fixed now.

haslinghuis reviewed (changes requested):
Still see a mix of links - with or without .md for example.
Some use relative doc path where others do not use any path

ctzsnooze commented:
Thanks @haslinghuis 
- have removed all .md extensions from file names in links
- replaced all instances of double slashes in links with single slashes
- make links to `https://betaflight.com/docs` into local links to `/docs/`, fixing local build image link loss
- found it was most reliable to use `/docs/` than `docs/` so did that consistently
- fixed what happens when clicking on say 'Tuning' category sidebar menu, the use of `/docs` makes links to `/docs/category/tuning` work, whereas `docs/category/tuning` would be unreliable or fail on build.

Builds with no errors, though Docusaurus appears very weak when it comes to detecting link errors, both locally with `yarn start` and when making startic files using `yarn build`.

chmelevskij commented:
> llink names must be capitalised the exact same way as the file name, or they won't work.

Guessing you are on MacOS/OSX? I've mentioned that before in configurator code as well, that we shoule be avoiding `TitleCase` `camelCase` etc. for files and agree on hyphens or underscores. By default it uses case insensitive filesystem, what makes it even worse than ppl start renaming things to just change the case of letters...
